.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -223,23 +232,35 @@ jobs:
       uses: ./.github/actions/download-artifact-with-retry
       with:
         name: nvd
-        path: nvd.zip
+        path: .
 
     - name: Build updater
       run: |
         echo "Building updater for pull request ${{ env.PR_NAME }}..."
         make tag
         make -C scanner bin/updater
 
+    - name: Sanity check NVD zip
+      run: |
+        if [ ! -f ./nvd.zip ]; then
+          echo "nvd.zip not found!"
```

Comment:
I thought `download-artifacts` would fail if the artifact did not exist. Is that assumption incorrect? I guess I want to understand if this is needed or if it's just you being extra careful.

Reply:
`download-artifacts` succeeded but the ouput folder name is `nvd.zip` so nvd.zip file itself cannot be found.

Reply:
OK, and the updater silently ignores the fact that the `STACKROX_NVD_ZIP_PATH` does not exist?

Reply:
It has logs says something like NVD is missing.

Reply:
```

***"level":"debug","component":"libvuln/updates/NewManager","bundle":"nvd","factory":"nvd","time":"2024-09-12T20:08:23Z","message":"factory unconfigurable"***
***"level":"warn","bundle":"nvd","component":"libvuln/updates/Manager.Run","error":"feed file is not a regular file: nvd.zip","updater":"nvd","time":"2024-09-12T20:08:23Z","message":"failed configuring updater, excluding from current run"***
***"level":"info","bundle":"nvd","component":"libvuln/updates/Manager.Run","total":0,"batchSize":4,"time":"2024-09-12T20:08:23Z","message":"running updaters"***
```

Reply:
But it doesn't fail, right?

Reply:
I think the exporter should fail if one of its updaters/enrichers fail to configure.

Reply:
We can capture that tech debt elsewhere, and for now let's keep the check.